src/util.js

```diff
@@ -8,6 +8,17 @@
 export function exprToString(
   expr: {[string]: any},
   top: boolean = true
+): string {
+  const strExpr = exprToStringInner(expr, top);
+  if (strExpr[0] === '(' && strExpr[strExpr.length - 1] === ')') {
+    return strExpr.substring(1, strExpr.length - 1);
+  } else {
```

Comment:
I know I wrote this, but I would add a brief comment here explaining what this code is doing.